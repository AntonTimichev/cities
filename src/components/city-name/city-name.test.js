import React from 'react';
import renderer from 'react-test-renderer';

import CityName from './city-name.jsx';

const mock = {
  cityName: `Amsterdam`,
  currentCity: `Hamburg`,
  onCityNameClick: jest.fn()
};

it(`CityName is rendered correctly`, () => {
  const {cityName, currentCity, onCityNameClick} = mock;
  const tree = renderer.create(<CityName
    cityName={cityName}
    currentCity={currentCity}
    onCityNameClick={onCityNameClick}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
