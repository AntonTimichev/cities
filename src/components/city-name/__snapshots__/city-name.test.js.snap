// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityName is rendered correctly 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item "
    href="#"
    onClick={[Function]}
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;
